src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+    @FindBy(css = "li a.songs")
+    WebElement allSongsBtn;
+   @FindBy (css = "[data-testid='sound-bar-play']")
+   WebElement soundBar;
+   @FindBy (css = "[data-testid='play-btn']")
+   WebElement playBtn;
+    @FindBy (css = ".side.player-controls" )
+    WebElement controlPanel;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```